ApiBundle/Transformer/NodeGroupRoleTransformer.php

```diff
@@ -5,28 +5,43 @@
 use OpenOrchestra\ApiBundle\Exceptions\HttpException\RoleNotFoundHttpException;
 use OpenOrchestra\ApiBundle\Exceptions\TransformerParameterTypeException;
 use OpenOrchestra\Backoffice\Collector\RoleCollectorInterface;
+use OpenOrchestra\BackofficeBundle\Model\GroupInterface;
 use OpenOrchestra\BackofficeBundle\Model\NodeGroupRoleInterface;
 use OpenOrchestra\BaseApi\Facade\FacadeInterface;
 use OpenOrchestra\BaseApi\Transformer\AbstractTransformer;
+use OpenOrchestra\BaseApi\Transformer\TransformerWithContextInterface;
```

Comment:
I would rather keep this interface in the ApiBundle because it is only related to this case.

Moreover, I would name it `TransformerWithGroupContextInterface` instead